bootstraptest/test_yjit.rb

```diff
@@ -2484,6 +2484,11 @@ def is_odd(obj)
   [is_odd(123), is_odd(456), is_odd(bignum), is_odd(bignum+1)]
 }
 
+# Float calls
+assert_equal '[2.0, 0.0, 1.0, 4.0]', %q{
+  [1.0 + 1.0, 1.0 - 1.0, 1.0 * 1.0, 8.0 / 2.0]
+}
```

Comment:
Your test is perfectly good and valid, but I will point out in case you don't know: numbers having binary fractions are also guaranteed to be representable exactly using floating-point numbers :)

e.g. `1.50`, `1.25`, `1.125`, `2.5` all have exact IEEE FP representations.

Reply:
I wasn't clever enough to figure out `1.125` is `2^0 + 2^(-3)` right away, but thanks for a nice refresher 🙂